src/Parser.js

```diff
@@ -6,12 +6,10 @@ import { ReactComponent as ArrowRight } from "./arrow-right.svg";
 import { ReactComponent as ArrowDown } from "./arrow-down.svg";
 
 // const startRegex = /section_start:(?<startTimestamp>\d+):(?<sectionName>.+)\re\[0K(?<sectionHeader>.+)?/;
-const startRegex = /section_start:(?<startTimestamp>\d+):(?<sectionName>.+)/;
+const startRegex = /section_start:(?<startTimestamp>\d+):(?<sectionName>[a-zA-Z0-9-_]+)(?:\[(?<sectionOptions>[a-zA-Z0-9-_,=]+)\])?/;
 
 // const endRegex = /section_end:(?<endTimestamp>\d+):(?<sectionName>.+)\re\[0K/;
-const endRegex = /section_end:(?<endTimestamp>\d+):(?<sectionName>.+)/;
-
-const { pathname } = window.location;
```

Comment:
This did no longer provide access to query strings under recent manifest version. The path was therefor now incorrect.